crates/nu_plugin_gstat/src/nu/mod.rs

```diff
@@ -27,6 +27,7 @@ impl SimplePluginCommand for GStat {
 
     fn signature(&self) -> Signature {
         Signature::build(PluginCommand::name(self))
+            .switch("disable-tag", "Disable git tag resolving", None)
```

Comment:
bikeshedding here but maybe `without-tag` or `no-tag`? I think `without-tag` is more descriptive.